server/server.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require('supertest');
+const app = require('./app');
+
+describe('Test the thumbnail API path path', () => {
+  test('It should response the an object with listingId and thumbNail', () => {
+    const expected = ['listingId', 'thumbNail'];
+    return request(app)
+      .get('/api/photos/thumbnail/0')
+      .then((response) => {
+        const keyValues = Object.keys(JSON.parse(response.text));
+        expect(typeof JSON.parse(response.text)).toBe('object');
+        expect(keyValues).toEqual(expect.arrayContaining(expected));
+      });
+  });
+});
+
```

Comment:
might be some typos here but the test looks good!

Reply:
The test passes i swear 😭!